src/labels/index.scss

```diff
@@ -2,3 +2,4 @@
 @import "./labels.scss";
 @import "./states.scss";
 @import "./counters.scss";
+@import "./keyboard-shortcuts.scss";
```

Comment:
Wasn't sure where to add this component? Creating a new top level component felt a bit overkill for such a "simple/small" component like this one. It's currently under "Labels".